kotlin/services/cloudwatch/Readme.md

```diff
@@ -1,30 +1,96 @@
+### Get started
 
-You will find these examples: 
+- [Hello Amazon CloudWatch](https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/kotlin/services/cloudwatch/src/main/kotlin/com/example/cloudwatch/HelloService.kt) (listMetricsPaginator command)
 
-- **CreateAlarm** - Demonstrates how to create an Amazon CloudWatch alarm.
-- **CreateSubscriptionFilter** - Demonstrates how to create an Amazon CloudWatch log subscription filter.
-- **DeleteAlarm** - Demonstrates how to delete an Amazon CloudWatch alarm.
-- **DeleteSubscriptionFilter** - Demonstrates how to delete Amazon CloudWatch log subscription filters.
-- **DescribeAlarms** - Demonstrates how to get information about Amazon CloudWatch alarms.
-- **DescribeSubscriptionFilters** - Demonstrates how to get a list of Amazon CloudWatch subscription filters associated with a log group.
-- **DisableAlarmActions** - Demonstrates how to disable actions on an Amazon CloudWatch alarm.
-- **EnableAlarmActions** - Demonstrates how to enable actions on a CloudWatch alarm.
-- **GetLogEvents** - Demonstrates how to get log events from Amazon CloudWatch.
-- **GetMetricData** - Demonstrates how to get Amazon CloudWatch metric data.
-- **PutEvents** - Demonstrates how to put a sample CloudWatch event.
-- **PutMetricAlarm** - Demonstrates how to create a new Amazon CloudWatch alarm based on CPU utilization for an instance.
-- **PutRule** - Demonstrates how to create an Amazon CloudWatch event-routing rule.
+### Single action
 
-To run these examples, you can setup your development environment to use Gradle to configure and build AWS SDK for Kotlin projects. For more information, 
-see [Get started with the AWS SDK for Kotlin](https://docs.aws.amazon.com/sdk-for-kotlin/latest/developer-guide/setup.html). 
+Code excerpts that show you how to call individual service functions.
+
+- [Create anomaly detector](https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/kotlin/services/cloudwatch/src/main/kotlin/com/example/cloudwatch/CloudWatchScenario.kt)  (putAnomalyDetector command)
+- [Create a dashboard](https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/kotlin/services/cloudwatch/src/main/kotlin/com/example/cloudwatch/CloudWatchScenario.kt)  (putDashboard command)
+- [Create a metric alarm](https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/kotlin/services/cloudwatch/src/main/kotlin/com/example/cloudwatch/CloudWatchScenario.kt)  (putMetricAlarm command)
+- [Delete alarms](https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/kotlin/services/cloudwatch/src/main/kotlin/com/example/cloudwatch/CloudWatchScenario.kt) (createKeyspace command)
+- [Delete an anomaly detector](https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/kotlin/services/cloudwatch/src/main/kotlin/com/example/cloudwatch/CloudWatchScenario.kt)  (DeleteAnomalyDetectorRequest command)
```

Comment:
The capitalization of DeleteAnomalyDetectorRequest is inconsistent with the others.